assignments/this.js

```diff
@@ -1,26 +1,55 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. When in the global scope or window, 'this' binds to the window/global scope
+* 2. Used with dot notation. Whatever is to the left of the dot (usually oints to object name) becomes 'this'.
+* 3. Used in conjuction with constuctor functions. 'This' refers ONLY to the instance of the new obj being created
+* 4. Explicitly defined whenever using the .call or .apply methods
 *
 * write out a code example of each explanation above
 */
 
 // Principle 1
+function myName(name) {
+    console.log(this);
```

Comment:
`this` is currently the `window`, but can you give me an example where something is bound to the window?